index.js

```diff
@@ -109,4 +138,14 @@ const stream = function (ctx, filepath, options) {
   return handleRequest(ctx, filepath, options);
 };
 
-module.exports = stream;
+const streamBuffer = function (ctx, buffer, contentType, options) {
+  assert(ctx, 'koa context required');
+  assert(buffer instanceof Buffer, 'buffer required');
+  options = options || {};
+  return handleBuffer(ctx, buffer, contentType, options);
+};
+
+module.exports = {
```

Comment:
I would rather export these as `file` and `buffer` and use it as:

```javascript
var stream = require('koa-stream')

app.use(function *() {
  yield stream.file(...);
});
```

What do you think?